pennylane/_qubit_device.py

```diff
@@ -210,7 +210,11 @@ def execute(self, circuit, **kwargs):
                 r = self.statistics(
                     circuit.observables, shot_range=[s1, s2], bin_size=shot_tuple.shots
                 )
-                r = qml.math.squeeze(r)
+
+                if type(r) == 'list':
+                    r = r[0]
+                else:
+                    r = qml.math.squeeze(r)
```

Comment:
Personally I don't like this addition, but it seemed to be required because when using jit the resulting array is wrapped in a list.